### PR TITLE
[skip email] More json->nwb fix

### PR DESCRIPTION
**Pull request by hanhou**
### Describe changes:
add float()

### What issues or discussions does this update address?
- https://github.com/AllenNeuralDynamics/dynamic-foraging-task/issues/271

### Describe the expected change in behavior from the perspective of the experimenter
None

### Describe the outcome of testing this update on a rig in 447
N/A

### CODE REVIEW

File: src/foraging_gui/TransferToNWB.py [hunk 1]
```diff
@@ -489,8 +493,6 @@ class obj:
     logger.setLevel(logging.DEBUG)
     logger.addHandler(logging.StreamHandler())
     
-    # bonsai_to_nwb(R'F:\Data_for_ingestion\Foraging_behavior\Bonsai\AIND-447-G1\668546\668546_2023-09-19.json')
-    
-    bonsai_to_nwb(R'F:\Data_for_ingestion\Foraging_behavior\Bonsai\AIND-447-1-D\707254\707254_2024-02-16_13-18-41\TrainingFolder\707254_2024-02-16_13-18-41.json')
+    bonsai_to_nwb(R'F:\Data_for_ingestion\Foraging_behavior\Bonsai\AIND-447-G1\668546\668546_2023-09-19.json')
```
Comment (alexpiet) [inline src/foraging_gui/TransferToNWB.py:496]: Whats the logic of hard-coding `AIND-447-G1` ? G1 is an outdated computer name. This code will also be run on computers outside of 447 (ephys room, for example)

Comment (hanhou) [reply src/foraging_gui/TransferToNWB.py:496]: This is just for local testing on my PC. When the file is plugged into a pipeline, the user should import and call `bonsai_to_nwb`.

**Comment by hanhou** [comment]
@alexpiet can you review this again?

**Review by alexpiet — APPROVED** [review]

**Comment by alexpiet** [comment]
How long does it take to run the NWB conversion? Can we just run it when the data is saved?

**Comment by hanhou** [comment]
> How long does it take to run the NWB conversion? Can we just run it when the data is saved?

Takes 2 seconds or so. Yes, we can, and I've been doing for a while. But Jerome advocates that we generate nwbs in Code Ocean, using this kind of pipeline https://cuddly-chainsaw-o4q9v84.pages.github.io/nwb/nwb_pipeline.html. All modalities, including behavior, are added to the `nwb_base_subject` one by one. Let's discuss this with him in the Hackathon.

**Pull request merged**